README.md

```diff
@@ -45,7 +45,76 @@ Host your own `amazon-connect-chat-interface.js` bundle file and provide the lin
 
 ![Host Widget Snippet Integrated Custom UI](/.github/screenshots/custom-chat-widget-interface-screenshot.png)
 
-### Option 3: Customized Widget and Chat Interface UI (Self-Hosted)
+### Option 3: Customize Widget Start Chat Response
+
+To get help setting up your own start chat backend and calling your own backend to start the chat contact instead of the Amazon-hosted backend, follow the instructions in the [startChatContactAPI](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/cloudformationTemplates/startChatContactAPI) section of the Git repository README to create an API to call the StartChatContact API. Once the API is created, you can call it in the CustomStartChat callback function and send the response back in the widget script.
+
+```diff
+<script type="text/javascript">
+    (function(w, d, x, id){s=d.createElement('script');s.src='https://${apiId}.cloudfront.net/amazon-connect-chat-interface-client.js';s.async=1;s.id=id;d.getElementsByTagName('head')[0].appendChild(s);w[x]=w[x]||function(){(w[x].ac=w[x].ac||[]).push(arguments)}})(window, document, 'amazon_connect', '360f3075-asfd-asfd-asdf-asdf');
+
+    amazon_connect('styles', { openChat: { color: '#ffffff', backgroundColor: '#07b62a'}, closeChat: { color: '#ffffff', backgroundColor: '#07b62a'} });
+    amazon_connect('snippetId', 'replace_with_snippetId');
+    amazon_connect('supportedMessagingContentTypes', [ 'text/plain', 'text/markdown' ]);
+    ....
++   amazon_connect('customStartChat', async function(callback) {
++         window.fetch('https://replace_with_your_StartChatContact_api_url', {
++            method: "POST",
++            body: JSON.stringify({
++                ConnectInstanceId: "replace_with_testing_instanceId",
```

Comment:
Can we mention here that the API payload does not always need to include these params? because this is called by client to customer backend, they can modify the payload and send any info they want to process on their backend